plugins/Toolbox/resources/qml/ToolboxCompatibilityChart.qml

```diff
@@ -163,23 +176,40 @@ Item
 
     Label
     {
-        id: technical_data_sheet
+        id: data_sheet_links
         anchors.top: combatibilityItem.bottom
         anchors.topMargin: UM.Theme.getSize("default_margin").height / 2
-        visible: base.technicalDataSheetUrl !== undefined
+        visible: base.technicalDataSheetUrl !== undefined ||
+            base.safetyDataSheetUrl !== undefined || base.printingGuidelinesUrl !== undefined
         height: visible ? contentHeight : 0
         text:
         {
+            var result = ""
             if (base.technicalDataSheetUrl !== undefined)
             {
-                return "<a href='%1'>%2</a>".arg(base.technicalDataSheetUrl).arg("Technical Data Sheet")
+                result += "<a href='%1'>%2</a>".arg(base.technicalDataSheetUrl).arg("Technical Data Sheet")
```

Comment:
It's not a new issue, but this text `Technical Data Sheet` needs to be translated.

Since 3.6 is string frozen, we'd need to change that separately in Master.

Reply:
Done here: https://github.com/Ultimaker/Cura/pull/4770